cordformation/src/main/resources/net/corda/plugins/runnodes

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -eo pipefail
 
```

Comment:
What does this do?

Reply:
Causes the shell to interpret variables differently.

Reply:
This one?
> Treat unset variables and parameters other than the special parameters "@" and "*" as an error when  performing parameter  expansion.   If  expansion is attempted on an unset variable or parameter, the shell prints  an  error message,  and, if not interactive, exits with a non-zero status.

Hmm, why remove this?

Reply:
because the expansion of JAVA_HOME fails when it is set.